torch/ao/nn/intrinsic/quantized/dynamic/modules/linear_relu.py

```diff
@@ -28,9 +30,15 @@ class LinearReLU(nnqd.Linear):
         torch.Size([128, 30])
     """
 
-    _FLOAT_MODULE = nni.LinearReLU  # type: ignore[assignment]
+    _FLOAT_MODULE = nni.LinearReLU
```

Comment:
Nit: this should actually be a ClassVar as annoying as that is to annoated properlty here:
```suggestion
    _FLOAT_MODULE : ClassVar[Type[nni.LinearReLU]] = nni.LinearReLU